components/brave_wallet/browser/blockchain_registry.cc

```diff
@@ -139,7 +138,7 @@ void BlockchainRegistry::GetTokenListMap(
             -> brave_wallet::mojom::BlockchainTokenPtr {
           return current_token.Clone();
         });
-    token_list_map_copy[key] = std::move(tokens_copy);
+    token_list_map_copy[chain_id] = std::move(tokens_copy);
   }
```

Comment:
My thought for using chain ID instead of the combined key created from `GetTokenListKey` was that when looping through the key, values in OnGetEthTokenRegistry the chain ID is accessible without trying to get it by decomposing the combined key.